src/shared/common/forms/radio-group.jsx

```diff
@@ -22,8 +24,29 @@ class Radio extends Component {
         }
     }
 
-    handleChange(e) {
+    handleChange(e, choice) {
         this.props.updateState({ [this.props.name]: e.target.value });
+        if ('conditionalContent' in choice) {
+            this.props.updateState({ [`${choice.value}Text`]: '' });
+        }
+    }
+
+    handleChangeForTextArea(e) {
+        this.props.updateState({ [e.target.id]: e.target.value });
+    }
+
+    isConditionalContentError(errors, contentLabelKey) {
+        if(errors && contentLabelKey in errors) {
+            return true;
+        }
+        return false;
+    }
```

Comment:
oh actually, if you felt fancy you could
```suggestion
    isConditionalContentError(errors, contentLabelKey) {
        return errors && contentLabelKey in errors
    }
```